### DISCUSSION TITLE
Doc comments

### DISCUSSION THREAD

**Discussion by euberdeveloper** [prisma/prisma#6704] [Q&A]
In addition to the api, I am developing an API client (that makes the requests to my api) in typescript and publishing it on npm.

I am using typedoc to document it and I was wondering if there was a way to add documentation comments to the Models. For instance if I add in the schema a comment before a property, then that comment is added also in the Typescript type definition so I got hints by vscode and typedoc parses that comment.

**Comment by janpio** [comment]
This is what I know about comments in Prisma Schema: https://www.prisma.io/docs/concepts/components/prisma-schema#comments Unfortunately that is not very clear, so I asked internally and will come back here when I hear something.

**Comment by euberdeveloper** [comment]
Thank you, I will try, maybe the triple-slashed comment works

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Reply by euberdeveloper** [reply]
The link doesn't brings anywhere, so I don't think this question has been answered. Maybe a collaborator could help?
